### DISCUSSION TITLE
Add TRIGGER WORDS box with the COPY button

### DISCUSSION THREAD

**Discussion by ostap667inbox** [BlafKing/sd-civitai-browser-plus#206] [Ideas]
When looking at information about a model or LoRA, we see many useful things. Many of them are not even necessary for everyday work, such as the model author's nickname and avatar, or a long list of license permissions. All of this can be viewed on the model page if needed.
But there is no real relevant and useful information when looking at information about LoRA: trigger words. 
![civ2](https://github.com/BlafKing/sd-civitai-browser-plus/assets/123593542/c6f30270-72a3-4ef8-93c1-8a88312a67cd)

It would be nice if the list of TRIGGER WORDS was also visible. A 'copy' button next to it would also be very useful.
![civ1](https://github.com/BlafKing/sd-civitai-browser-plus/assets/123593542/3de450a9-07e3-45e5-910a-25cf915297e4)

Also, the info lacks another useful info that the authors add to the description: USAGE TIPS. This could be very useful information, such as LoRA strength or Clip Skip.
![civ3](https://github.com/BlafKing/sd-civitai-browser-plus/assets/123593542/d1024488-d3d7-4160-881b-a6daab1f04ce)

**Comment by BlafKing** [comment]
Sounds good!

On the browser the trigger words are included in the trained tags box:
![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/9644716/3bc64fc6-69ff-4779-a937-c290a3b3868e)


But I forgot that these wouldn't be displayed if you click the CivitAI button on the model cards, I'll a text box with the triggerd words on the model page as well 👍🏻 

About the usage tips, I don't think these can be added as of right now since the public CivitAI API does not return these values in the response.
So this would have to be added by CivitAI's devs first before I can add it to the extension.

I also recommend updating the model info & tags for all your models from the update tab, this will make the trigger words automatically get inserted whenever you select a LORA!

**Comment by ostap667inbox** [comment]
_> I also recommend updating the model info & tags for all your models from the update tab, this will make the trigger words automatically get inserted whenever you select a LORA!_

This advice won't always produce a good result. Very often in LoRA descriptions we have to add additional words ourselves so that they are automatically inserted into the prompt.
For example, LoRA has the trigger words "by Hans Higer", but in the description the author writes that adding the words "illustration, horror art" to the prompt makes the result even better. These are not trigger words, but we often manually add them to this field in the model description so that they are automatically added to the prompt as well.
Therefore, automatic updating of all model info may not be desirable.
